src/OpinionatedVestingWalletFactory.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./BaseVestingWalletFactory.sol";
+
+/**
+ * @title OpinionatedVestingWalletFactory
+ * @dev Factory for VestingWalletWithCliffAndClawback. See VestingWalletWithCliffAndClawback.sol for details.
+ *      This factory creates wallets with 4-year vesting and a 1-year cliff.
+ */
+contract OpinionatedVestingWalletFactory is BaseVestingWalletFactory {
+
+    /**
+     * @dev Creates a new VestingWalletWithCliffAndClawback with 4-year vesting and 1-year cliff.
+     */
+    function create(
+        address ownerAddress,
+        address beneficiaryAddress,
+        uint64 startTimestamp
+    ) public virtual returns (address) {
+        return _create(
+            ownerAddress,
+            beneficiaryAddress,
+            startTimestamp,
+            4 * 52 weeks, // `years` and `months` are deprecated
+            52 weeks
```

Comment:
```suggestion
            4 * 365 days, // `years` and `months` are deprecated
            365 days
```

Reply:
52 * 7 = 364 🤷 
If you want to get really nit-picky and talk about leap years, we could make the duration 4 * 365 days + 1 days.